src/index.ts

```diff
@@ -1,3 +1,3 @@
 export { getManagementToken } from './keys'
 export { signRequest, verifyRequest, ContentfulHeader } from './requests'
-export { Secret, CanonicalRequest, Timestamp, SignedRequestHeaders } from './requests/typings'
+export { CanonicalRequest, SignedRequestHeaders } from './requests/typings'
```

Comment:
Why?

Reply:
Plus, this should be moved into the `./requests/index.ts` and exported from there

Reply:
Secret and Timestamp are just `string` and `number` types, and they are not otherwise relevant to the user, so it make little sense to export them.
I agree that they should be exported through `requests/index`